src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/Binding.Custom.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <None Include="project.json" />
   </ItemGroup>
   <ItemGroup>
+     <TestCommandLines  Condition="'$(TargetOS)'!='Windows_NT'" Include="sudo ./InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file /tmp/wcfrootca.crt"/>
```

Comment:
sudo needs to be passed with -E -n, i.e., something like 

`"sudo -E -n ./InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file /tmp/wcfrootca.crt"`